providers/database_my_sql.rb

```diff
@@ -22,12 +22,26 @@ def load_current_resource
 def database_exists?(dbname, host, port, root_username, root_password)
   exists = false
   db = nil
+  connection = false
   begin
     db = ::Mysql.new(host, root_username, root_password, dbname, port)
+    connection = true
+    # Make sure the last query is completed for database population
+    rs = db.query("SELECT sysmap_shapeid FROM sysmap_shape LIMIT 1")
+    if rs.num_rows < 1
+      raise ::Mysql::Error
+    end
     exists = true
     Chef::Log.info("Connection to database '#{dbname}' on '#{host}' successful")
   rescue ::Mysql::Error
-    Chef::Log.info("Connection to database '#{dbname}' on '#{host}' failed")
+    if connection
+      Chef::Log.info("Database '#{dbname}' popualtion incomplete, clean database and re-run")
```

Comment:
`population` spelling

Reply:
Addressed comment, and added config for php-fpm cookbook so we don't need to fork that repo